documentation/staging/content/release notes/_index.md

```diff
@@ -26,4 +28,4 @@ pre = "<b> </b>"
 
 - The application is limited to working with archive files whose size is less than 2 GB.
 
-- On MacOS, after editing a model in the Model Code View's model editor pane, you must click outside the editor pane prior to using the Menu to save or perform other actions that need the latest model content.  This is due to Electron bug [31915](https://github.com/electron/electron/issues/31915) that fails to generate the proper events when the Menu is selected on MacOS.
+- On macOS, after editing a model in the Model Code View's model editor pane, you must click outside the editor pane prior to using the Menu to save or perform other actions that need the latest model content.  This is due to Electron bug [31915](https://github.com/electron/electron/issues/31915) that fails to generate the proper events when the Menu is selected on macOS.
```

Comment:
We can remove this bullet point since we found a way to work around it that no longer requires the user to do anything...